recipe/meta.yaml

```diff
@@ -27,9 +27,9 @@ requirements:
     - rtree
     - descartes
     - matplotlib
-    - psycopg2
+    # - psycopg2
```

Comment:
@jorisvandenbossche if my findings are correct `psycopg2` is the culprit b/c `conda-forge` is in the middle of the `openssl` migration and `conda` ends up preferring an old `libgdal` from `defaults` instead of the `libgdal 2.4` from `conda-forge`. We can:

- remove `psycopg2` for now, it is an optional dependency, and re-add it later;
- force `libgdal 2.4` here. bu over-specifing our dependencies we help conda do the right thing;
- or implement the `strict` channel policy on the feedstocks. (Pinging @conda-forge/core here for this one.)

@msarahan the issue on the staged-recipe package you pinged me a few days ago is probably similar. Sorry but I lost the link and I forgot the name of the package.

Reply:
https://github.com/conda-forge/staged-recipes/pull/7871

Reply:
Thanks! Looking into it now to see if it is the same issue. I'm afraid that the best solution would be enforcing `strict` channel in our builds.

Reply:
I think that's a good solution.  I think conda should consider changing to strict channel priority by default for conda 4.7, but before then I'd like to try to improve the messaging around unsatisfiable dependencies.  A lot of people have been confused about how strict channel priority makes some packages unavailable, even though they seem to be available by inspection.

Reply:
I don't understand the conda forge pinning enough to know if forcing libgdal to 2.4 could cause problems (if it is pinned like that, why do we actually not get it? Is it possible right now to install geopandas with eg gdal 2.3?). 
So I would defer to your judgement. Removing psycopg2 is fine (then also sqlalchemy). That might be a good thing anyway, as it is quite optional.

Reply:
> I don't understand the conda forge pinning enough to know if forcing libgdal to 2.4 could cause problems (if it is pinned like that, why do we actually not get it?

It won't cause a lot of problems but we won't be able to install `geopandas 0.4.1` with `libgdal 2.3`.

>  Is it possible right now to install geopandas with eg gdal 2.3?).

It is as long as you use the `strict` channel :smile: 

> So I would defer to your judgement. Removing psycopg2 is fine (then also sqlalchemy). That might be a good thing anyway, as it is quite optional.

Let's go for this option then. Do you want to also remove `sqlalchemy` for consistently not provide the optional deps?

Reply:
> Do you want to also remove sqlalchemy for consistently not provide the optional deps?

just because the one is only useful with the other